_layouts/project.html

```diff
@@ -4,10 +4,162 @@
     {% include  assets.html %}
+    <link rel="stylesheet" href={{ "/css/project.css" | relative_url }} />
   </head>
   <body>
-    {% include header.html %}
-    <div class="page-content">{{ content }}</div>
-    <script type="text/javascript" src={{ "/js/project_processor.js" | relative_url }}></script>
+    <nav>
+      <div class="nav-wrapper blue">
+        <a href="/" class="brand-logo">NITDgpOS</a>
+        <ul id="nav-mobile" class="right hide-on-med-and-down">
+          <li><a href="https://github.com/nit-dgp/{{page.title}}">Get Involved!</a></li>
+        </ul>
+      </div>
+    </nav>
+    <div class="center main-content">
+      {% if page.logo %}{% endif %}
+      <h1 class="title"><a id="main-title">{{page.title}}</a><h1>
+      <h2 class="flow-text description black-text text-lighten-2">{{page.description}}<h2>
+    </div>
+    <hr class="container" />
+
+    {% if page.gallery %}
+      <div class="container gallery">
+        <h2 class="center title">Gallery</h2>
+        <div class="row">
+          {% for image in page.gallery %}
+            <div class="col s12 m6">
+              <div class="card">
+                <div class="card-image hoverable">
+                  <img data-caption="{{ image.title }}" class="materialboxed" src="{{ image.url }}">
+                  <span class="card-title">{{ image.title }}</span>
+                </div>
+              </div>
+            </div>
+          {% endfor %}
+        </div>
+      </div>
+      <hr class="container" />
+    {% endif %}
+
+    <div class="container page-content">{{ content }}</div>
+    <hr class="container" />
+
+    <div id="stats" class="center container">
+      <h1 class="heading">Hello there!</h1>
+      <h4>We are the contributors of <b>{{page.title}}</b>.</h4>
+      <div id="contributors" class="row"></div>
+
+      <hr />
+      <h3 class="title">Stats, stats everywhere!</h3>
+      <p class="flow-text">written mostly in <b id="language"></b></p>
+      <p class="flow-text">
+        Help us <a id="issues-url">fix issues</a>.
+        We have <b id="issues-count"></b> open issues right now.
+      </p>
+      <p class="flow-text">We were forked <b id="forks-count"></b> times, in exact.</p>
+      <p class="flow-text"><a id="subscribe">Subscribe us</a> for more updates!</p>
+      <p class="flow-text">maintained by <a id="owner"></a></p>
+    </div>
+
+    <footer class="page-footer transparent">
+      <div class="container center">
+        <p class="grey-text license">LICENSED UNDER GNU AFFERO GENERAL PUBLIC LICENSE V3.0</p>
+      </div>
+      <div class="footer-copyright transparent">
+        <div class="grey-text lighten-4 container center">
+          © <b id="created"></b>-<b id="updated"></b> Copyright <a href="/">NITDgpOS</a>
+        </div>
+      </div>
+    </footer>
+
+    <script>
+    // Load project data from api.github.com
+
+    if("undefined" !== typeof jQuery) {
```

Comment:
You can use coffeescript as [mentioned here](https://jekyllrb.com/docs/assets/). I think it is fine for now though.